### PR TITLE
Homework8: Anastasiya Solodukho

### PR DESCRIPTION
**Pull request by AnastasiyaSo**
Added HW#8

### CODE REVIEW

File: homework_8.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+"""My Homework 8"""
+# "Task #1"
+
+
+def solution(*args):
+    """Check an increasing sequence"""
+    numbers = args[0]
```
Comment (Stml89) [inline homework_8.py]: you have already passed to the function list, no need to use `*args`

File: homework_8.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    index_number = 0
+    numbers_of_removing = 0
+    for i in numbers:
```
Comment (Stml89) [inline homework_8.py]: for i in range(len(numbers))

File: homework_8.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+print(solution([1, 2, 3, 4, 5, 3, 5, 6]))
+print(solution([40, 50, 60, 10, 20, 30]))
+
+# "Task #2"
+
+
+def find_opposite_number(n=10, first_number=2):
+    """Find opposite number"""
+    quantity_of_numbers = len(list(range(0, n)))
+    opposite_number = int(quantity_of_numbers / 2 + first_number)
```
Comment (Stml89) [inline homework_8.py]: opposite_number = int(n / 2 + first_number) ?

File: homework_8.py
Comment (Stml89) [inline homework_8.py:1]: Can not find task#3

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework_8.py (hunk 2 above)
Comment (AnastasiyaSo) [reply homework_8.py]: I don't understand why I need to use "range(len(numbers))"? I need to iterate by value not index

File: homework_8.py
Comment (AnastasiyaSo) [reply homework_8.py:1]: Because I couldn't find task in docs :) It will be done

**Comment by AnastasiyaSo** [comment]
Fixed. Please, check

File: homework_8.py [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            sum_numbers += n
+        index_number += 1
+
+    if sum_numbers % 10 == 0:
+        return True
+
+    return False
+
+
+assert validate(4561261212345467) is True
```
Comment (Stml89) [inline homework_8.py]: use `==` instead of `is`
https://www.geeksforgeeks.org/difference-between-and-is-operator-in-python/

File: homework_8.py [hunk 5]
```diff
@@ -0,0 +1,65 @@
+"""My Homework 8"""
+
+# "Task #1"
+
+
+def is_increasing_sequence(sequence):
+    """Check an increasing sequence"""
+    sequence = tuple(sequence)
```
Comment (Stml89) [inline homework_8.py]: `sequence` variable comes as a list, you do not need to convert to tuple or any other type. Work with list

**Review by Stml89 — CHANGES_REQUESTED** [review]

**Comment by AnastasiyaSo** [comment]
@Stml89, fixed

File: homework_8.py (hunk 2 above)
Comment (Stml89) [reply homework_8.py]: if it is more convenient for you, no problem from me side

**Review by Stml89 — APPROVED** [review]
You can merge the PR

**Pull request merged**